app/forms/DepartureDetails.scala

```diff
@@ -16,18 +16,27 @@
 
 package forms
 
-import forms.common.Date
-import play.api.data.Forms
+import java.time.{LocalDateTime, LocalTime}
+
+import forms.common.{Date, Time}
+import play.api.data.Forms._
+import play.api.data._
 import play.api.libs.json.{Json, OFormat}
 
-case class DepartureDetails(dateOfDeparture: Date) {
+case class DepartureDetails(dateOfDeparture: Date, timeOfDeparture: Time) {
+
+  def moment: LocalDateTime = LocalDateTime.of(dateOfDeparture.asLocalDate, timeOfDeparture.time)
```

Comment:
Could you provide some more explanatory name? Without a context it is hard to say what it actually does.

Reply:
departure moment  - as DepartureDetails#moment